### DISCUSSION TITLE
GitHub Copilot doesn't show up and not working in VSC

### DISCUSSION THREAD

**Discussion by PhilipBright** [community/community#77189] [Copilot Conversations]
i have paid for the GitHub copilot and I only used it for 3 days and it doesn't working at all, i have already tried uninstalling and revoking, didn't fix the error, can anyone help me?

**Comment by ghost** [comment]
เมื่อ 28 พ.ย. 2023 20:59 น. "CSPhilip" ***@***.***> เขียนว่า

i have paid for the GitHub copilot and I only used it for 3 days and it
doesn't working at all, i have already tried uninstalling and revoking,
didn't fix the error, can anyone help me?

—
Reply to this email directly, view it on GitHub
<https://github.com/orgs/community/discussions/77189>, or unsubscribe
<https://github.com/notifications/unsubscribe-auth/A57FFUJKIWF2FMTZ5P52J4DYGXU3LAVCNFSM6AAAAAA75ZXDG2VHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZVHEYDCMJYGY>
.
You are receiving this because you are subscribed to this thread.Message
ID: ***@***.***>

**Comment by fellan1996** [comment]
I have the same problem

**Reply by PhilipBright** [reply]
I tried to refund it and only refund me 1$, not worth for me. I haven't use that at all, always looking for the answer and spent days without using!

**Comment by lacostenycoder** [comment]
I had a personal account. My company bought me a license. Now I can't use the VSCode extention. It just disappears. I tried uninstall, reinstall and use edge version... nothing.
